src/idl/src/tree.c

```diff
@@ -2493,7 +2493,7 @@ static void *iterate_annotation(const void *ptr, const void *cur)
   const idl_node_t *node = cur;
   assert(root);
   if (node) {
-    assert(idl_parent(root) == node);
+    assert(idl_parent(node) == root);
```

Comment:
Ah, this probably saves me some work. Noticed a bug yesterday that may be caused by this :+1: